awx_collection/plugins/modules/tower_workflow_job_template.py

```diff
@@ -208,6 +224,34 @@ def main():
         organization_id = module.resolve_name_to_id('organizations', organization)
         search_fields['organization'] = new_fields['organization'] = organization_id
 
+    # Attempt to look up workflow job template to copy based on the provided name
+    if copy_from:
+        # Check if workflow job template exists, as API will allow you to create an identical item with the same name in same org, but GUI will not.
+        workflow_job_template = module.get_one('workflow_job_templates', name_or_id=name, data=search_fields)
+        if workflow_job_template is not None:
+            module.fail_json(msg="A workflow job template with the name {0} already exists.".format(name))
+        else:
+            # Lookup existing workflow job template.
+            copy_from_lookup = module.get_one('workflow_job_templates', name_or_id=copy_from)
+            if copy_from_lookup is None:
+                module.fail_json(msg="A workflow job template with the name {0} was not able to be found.".format(copy_from))
+            else:
+                workflow_copy_get_check = module.get_endpoint(copy_from_lookup['related']['copy'])
+                if workflow_copy_get_check['status_code'] in [200]:
+                    if (workflow_copy_get_check['json']['can_copy'] and workflow_copy_get_check['json']['can_copy_without_user_input'] and
+                        not workflow_copy_get_check['json']['templates_unable_to_copy'] and not workflow_copy_get_check['json']['credentials_unable_to_copy']
+                            and not workflow_copy_get_check['json']['inventories_unable_to_copy']):
+                        # Because the initial copy will keep its organization, this can be different then the specified one.
```

Comment:
In this instance you are checking the can_copy response from the copy endpoint but you don't for other objects. Perhaps this can become common if the code is centralized in copy_item?

Reply:
In this case, only the workflows have this check. 
[project copy endpoint](https://docs.ansible.com/ansible-tower/latest/html/towerapi/api_ref.html#/Projects/Projects_projects_copy_list)
[workflow copy endpoint](Fhttps://docs.ansible.com/ansible-tower/latest/html/towerapi/api_ref.html#/Projects/Projects_projects_copy_list)
If the can_copy options were added to other endpoints moving this to the method would be good.